pkg/protocol/args.go

```diff
@@ -177,6 +177,19 @@ func releaseArg(h []argsKV) []argsKV {
 	return h[:len(h)-1]
 }
 
+func updateArgBytes(h []argsKV, key, value []byte) []argsKV {
```

Comment:
What's this function for? Update arg but only process the case that `kv.noValue == true`? 
Caller may not get the outcome(whether updated or not) from the returning value since it always return `h`.